app/dao/jobs_dao.py

```diff
@@ -108,6 +109,8 @@ def dao_get_future_scheduled_job_by_id_and_service_id(job_id, service_id):
 
 
 def dao_create_job(job):
+    if not job.id:
+        job.id = uuid.uuid4()
```

Comment:
Yeah just wondering if there's anywhere else currently where this path would be hit, in other words if `create_letter_api_job` is the only place we need it then makes sense to do there?